src/compas_robots/viewer/scene/robotmodelobject.py

```diff
@@ -52,12 +51,18 @@ def __init__(
         self.hide_coplanaredges = hide_coplanaredges
         self._show_visual = show_visual or True
         self._show_collision = show_collision or False
-        self.configuration: Configuration = configuration or model.zero_configuration()
-        super(RobotModelObject, self).__init__(model=model, **kwargs)
+        self.configuration: Configuration = configuration
```

Comment:
why remove the default `zero_configuration`? also where is this used if a configuration needs to be sent when calling `update()`>

Reply:
was for debuging for got to put it back, thanks